pages/cards/_card.vue

```diff
@@ -48,9 +48,13 @@ import MetroCard from '@/components/cards/MetroCard.vue'
 import AgencyCard from '@/components/cards/AgencyCard.vue'
 /* eslint-enable simple-import-sort/sort */
 
+import { Vue, Component } from 'nuxt-property-decorator'
+import type { NuxtOptionsHead as MetaInfo } from '@nuxt/types/config/head'
+import { NuxtConfig } from '@nuxt/types'
```

Comment:
型のみのインポートなので、import type文にできそうです。

```suggestion
import type { NuxtConfig } from '@nuxt/types'
```